### PR TITLE
Moebius-first-week-journal-file

### PR DESCRIPTION
**Pull request by melrodbos**

### CODE REVIEW

File: Moebius-the-journal [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+- Moving around github (things that I understand (now))
+ - creating, issues, forking a repo, cloning a repo and creating pull requests
+ - The command line (git):
+   - Wow... Never thought I would use it or even understand it, but now I really appreciate managing almost everything in one place.
+   - Need to try to create a pull request from Terminal. 
+   - Working on a git cheat sheet.
+
+#### Week One
+
+So I though leaving my kids at home after three and half years of literally spending every second with them was going to be the difficult part… Not. The first week at the Iron Yard has been challenging to the bone. I have fallen asleep with my eyes open, read books an articles considerably above my level of understanding (I am humbled, like, really humbled) and now I have come to believe that the world is conspiring against any possible speck of new information wanting to crawl into my brain.
```
Comment (jorgehjr84) [inline Moebius-the-journal]: Line 1 , though should be changed to thought

Line 5, books 'and' articles

**Comment by jorgehjr84** [comment]
Awesome 1st journal entry Melissa! It was funny and you were very candid. Can't wait to read the next entry.

:+1:

**Comment by kmattiko** [comment]
Nice expressions of how challenging this process can be. Good Job :+1:

**Comment by al-the-x** [comment]
You need to merge this Pull Request before you get much farther. That _may_ cause conflicts with your in-progress PR, but we can resolve that with all that merge practice you're getting... or we can do it together.

**Pull request merged**